tests/functional/test_ec2.py

```diff
@@ -133,3 +133,4 @@ def test_copy_snapshot_injects_presigned_url(self):
             'a94a6b52afdf3daa34c2e2a38a62b72c8dac129c9904c61aa1a5d86e38628537'
         )
         self.assertEqual(query_args['X-Amz-Signature'], [expected_signature])
+
```

Comment:
The easiest way to resolve this may to just reset the commit and remove your first change to this. Using `git add -p <filename or directory>` will let you choose which lines you commit and avoid issues like this.